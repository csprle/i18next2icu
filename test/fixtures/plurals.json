{
  "item_zero": "No items",
  "item_one": "{{count}} item",
  "item_other": "{{count}} items"
}
