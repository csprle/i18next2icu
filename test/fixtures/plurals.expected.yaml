item: '{count, plural, =0{No items} one{{count} item} other{{count} items}}'
